src/arduino-rfm/LoRaMAC.cpp

```diff
@@ -616,21 +730,25 @@ bool LORA_join_Accept(sBuffer *Data_Rx,sLoRa_Session *Session_Data, sLoRa_OTAA *
 				//Clear Data counter
 				Data_Rx->Counter = 0x00;
 
-#ifdef DEBUG
-				Serial.print("NwkSKey: ");
-				for(byte i = 0; i < 16 ;++i)
-					Serial.print(Session_Data->NwkSKey[i],HEX);
-				Serial.print("\nAppSKey: ");
-				for(byte i = 0; i < 16 ;++i)
-					Serial.print(Session_Data->AppSKey[i],HEX);
-				Serial.println();
-#endif	
+	#ifdef DEBUG
+					Serial.print("NwkSKey: ");
+					for(byte i = 0; i < 16 ;++i)
+						Serial.print(Session_Data->NwkSKey[i],HEX);
+					Serial.print("\nAppSKey: ");
+					for(byte i = 0; i < 16 ;++i)
+						Serial.print(Session_Data->AppSKey[i],HEX);
+					Serial.println();
+	#endif	
 				joinStatus = true;
 			}
 		}
 	}
 	return joinStatus;
 }
 
+void LORA_Set_Interrupt(void){
+    pinMode(RFM_pins.DIO0, INPUT);
+    RFM_Write(0x12, 0x00); //REG_DIO_MAPPING_1 RX done
```

Comment:
Add new constants names "RFM_REG_IRQ_FLAGS"

Reply:
Almost all the registers are used without variable names

Reply:
Did you created names for the registers hex values? I have already created most of them in my repo and don't know if it's better to create a pull request now or wait until you merge this one.

Reply:
The better way should be by doing a pull request so you add all the registers numbers